### PR TITLE
[#418] testing reminder to update IRODS_VERSION

### PR DESCRIPTION
**Pull request by d-w-moore**
(also, get irods.test.helpers to load under MSWindows.)

### CODE REVIEW

File: irods/test/helpers.py
Comment (trel) [inline irods/test/helpers.py:109]: "is compatible up to" -> "advertises compatibility up to"

Comment (trel) [inline irods/test/helpers.py:109]: use all the letters?

"connected_vsn" -> "connected_version"
and
"advertised_vsn" -> "advertised_version"

Comment (d-w-moore) [reply irods/test/helpers.py:109]: Yes, thanks.  I was struggling there ...

**Comment by korydraughn** [comment]
Why is the build hook being removed? I feel I may have missed the reason.

**Comment by d-w-moore** [comment]
> Why is the build hook being removed? I feel I may have missed the reason.

Admittedly a judgement call, but it's no longer used.

**Comment by korydraughn** [comment]
If nothing is using it, seems fine to remove.

Does it contain anything important that we should capture somewhere?

**Comment by d-w-moore** [comment]
> Does it contain anything important that we should capture somewhere?

References to `python-coverage`, which appears to have been a package and a command (or script).
If it helps, I can split out the commit in which the build hook was removed, to make it more obvious....

**Comment by d-w-moore** [comment]
Other than that one thing, this appears to be a copy-paste build hook.  Not that the PRC needs anything to be actually built.

**Comment by alanking** [comment]
I am in favor of removing the build hook, but I do agree that keeping it in a separate commit for visibility would be better.

**Comment by d-w-moore** [comment]
> I am in favor of removing the build hook, but I do agree that keeping it in a separate commit for visibility would be better.

Done and done.

**Comment by d-w-moore** [comment]
squashed

**Review by alanking — APPROVED** [review]
Please await one more approval and I think we can add #s

**Review by trel — APPROVED** [review]

**Comment by d-w-moore** [comment]
squashed and slightly elaborated the commit messages.

**Review by korydraughn — APPROVED** [review]
If the tests pass and this operates as you expect, then I think it's safe to add the pounds.

**Review by alanking — COMMENTED** [review]
The build hook removal is now squashed into the base commit... Was that on purpose?

**Comment by d-w-moore** [comment]
I can pull it out again.  The issue would be packaging, correct?

**Comment by alanking** [comment]
Well, if everybody is okay with it, it's fine. Just wanted to make sure since earlier in this PR thread we had agreed to pull it out for reasons of "visibility":
https://github.com/irods/python-irodsclient/pull/419#issuecomment-1353223405
> > I am in favor of removing the build hook, but I do agree that keeping it in a separate commit for visibility would be better.
> 
> Done and done.

Thoughts, @korydraughn / @trel?

**Comment by d-w-moore** [comment]
Yep, agree it's better for it to be on its own.  Working it now ...

**Comment by d-w-moore** [comment]
Ok .  holding off # until folks have had a chance to look at the commit message.

**Review by alanking — APPROVED** [review]
Looks good to me. Maybe wait for one more approval before we add #s

**Review by korydraughn — APPROVED** [review]
Looks ready to me.

**Comment by alanking** [comment]
Let's # it

**Comment by d-w-moore** [comment]
ok , #'d .

**Pull request merged**
